### DISCUSSION TITLE
Waydroid on actual RHEL? Anyone tried? Successfully?

### DISCUSSION THREAD

**Discussion by AlexDeGruvenB** [waydroid/waydroid#1427] [Q&A]
I'm *attempting* to standardize on RHEL for my daily driver desktop (I'm a professional RHEL toucher by trade) as I migrate more of my workload off of Windows (partially because of my profession, partially for the same reasons as a lot of people recently).

I've used it on Fedora and it works well, but there's not an install available in the RHEL repos, and it doesn't appear that they're in RPM Fusion, either (at least not in the RHEL9 repos). I also have EPEL enabled but still coming up blank.

Anyone been able to get this going? I don't necessarily want to roll back to Fedora (even though I like the experience), and would rather stick on this if I can.

**Comment by Runa-Chin** [comment]
Well, actually RHEL (Red Hat Enterprise Linux) is based on older version of Fedora and I think it doesn't work. Even you ran under container, it's still doesn't work. So, you need to be patient for waiting RHEL to use base from Fedora 37 or up. Or, instead you can use openSUSE Slowroll. I have guide for that.

**Reply by x86-64D** [reply]
It should now, rhel10 based on newer fedora so
